packages/terra-tabs/src/common-tabs/_Tabs.jsx

```diff
@@ -75,12 +104,16 @@ const propTypes = {
 };
 
 class Tabs extends React.Component {
+  addTabId = null;
+
+  addButtonToggle = false;
```

Comment:
why do we need a variables here ?? Can't we make use of props `onSelectAddButton` to validate.

Reply:
Done https://github.com/cerner/terra-framework/commit/628052153b88461ef88733be6b05c1fb519e3f41